### ISSUE TITLE
Edit content field and remove type field from group.yml credits file

### ISSUE THREAD

**Issue by tamara-snyder** [hackforla/website#2833]
### Prerequisite
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started).
2. Please make sure you have read our **[Hack for LA Contributing Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md)** before you claim/start working on an issue.

### Overview
As a developer, I want to edit the content field and remove the type field from a credit's yml file so that redundant code is removed and the code is easier to understand.

### Details
Currently, in each credit's yml file, there is a `content` field and a `type` field, which contains similar information. Since the fields are redundant, we decided to remove the `type` field.  

Also, we changed the `content` field to the `content-type` field to make it clearer what its purpose is. The possible values for `content-type` field are image, video, or audio; other types of content can be added if necessary. In the future, this will allow developers a way to differentiate easily between different types of content in order to show each credit's media file correctly on the website.

### Action Items
For the file `_data/internal/credits/group.yml`, do the following:
- [x] Change line 4 for the content field from
  ```
  content: icon
  ```
  to
  ```
  content-type: image
  ```
- [x] Remove line 11 for the type field
  ```
  type: icon
  ```
- [ ] Once the pull request associated with this issue is approved and merged, please update and edit epic #2775 by
  - [ ] Checking off the dependency for this issue
  - [ ] If all dependencies are checked off, please move issue #2775 to the New Issue Approval column and remove the Dependency label

### Resources/Instructions
- This issue is tracked in the epic #2775.

#### File and Code links you will need to work on this issue
- Directory to find the page in once you have it in your IDE: `_data/internal/credits/group.yml`

- Click on the links below to see the source file code:
https://github.com/hackforla/website/blob/2443b6c47df54ca678a432035beb93f1e2e3b94e/_data/internal/credits/group.yml#L4
https://github.com/hackforla/website/blob/2443b6c47df54ca678a432035beb93f1e2e3b94e/_data/internal/credits/group.yml#L11

**Comment by sonalinajera** [comment]
ETA: End of day on 9/25
Availability: 2hrs

**Comment by kathrynsilvaconway** [comment]
Hi, this issue has not been updated in over 2 weeks. If you are still interested in working with HfLA, please let us know by updating your ETA and Availability. If we don't hear back within 7 days, we will unassign you from this issue and put it back in the queue. Thanks!

**Comment by JijiTheCreator** [comment]
Availability: 2hrs
ETA: 01/15/22

**Comment by ExperimentsInHonesty** [comment]
@JijiTheCreator Have you been able to drop into office hours or another dev meeting to get help?

**Comment by JijiTheCreator** [comment]
Hi, my apologies. Yes I dropped in last Thursday of January. I have been away since then and will begin working on workflow. If any further questions arise I will attend this upcoming Thursday meeting as well. Thank you.

**Comment by JijiTheCreator** [comment]
Availability: 2hrs
ETA: 02/08/22

**Comment by ExperimentsInHonesty** [comment]
@JijiTheCreator are you having trouble setting up your dev environment?  If so, could you provide some specifics about where you are stuck.

**Comment by jdingeman** [comment]
Reached out to dev regarding issue and he has time constraints due to a recent move. Unassigning him and putting back into the backlog so it can be picked up.

**Comment by vincentdang** [comment]
hey I don't think this will take too long.

i. Availability: night time on weekdays or weekends. (6-7pm and onward)
ii. ETA: in 2-3 days

**Comment by vincentdang** [comment]
update: I changed line 4 and deleted line 11.

**Comment by ExperimentsInHonesty** [comment]
@vincentdang Please change the name of the PR to `edit content field and remove type field from group.yml credits file` so we can accept it.
